skrifa/src/string.rs

```diff
@@ -597,12 +597,70 @@ const LANGUAGE_ID_TO_BCP47: &[(u16, &str)] = &[
     (0x540a, "es-US"),        //Spanish
 ];
 
+fn get_best_name(font: &FontRef, name_ids: Vec<StringId>) -> Option<String> {
```

Comment:
Any reason not to take an `impl IntoIterator<Item = StringId>`?

Reply:
I am happy to improve it if there is a chance for a merge.